src/main/java/com/code_intelligence/jazzer/mutation/engine/SeededPseudoRandom.java

```diff
@@ -73,6 +73,16 @@ public <T> int otherIndex(T[] array, int currentIndex) {
     }
   }
 
+  @Override
+  public int otherIndex(int length, int currentIndex) {
```

Comment:
I missed this during the merges, but @bertschneider suggested renaming `index` to `indexIn`. For the sake of consistency and clarity, we could also rename `otherIndex` (both overloads) to `otherIndexIn` and rename the first parameter to `range`.

Reply:
Done!